integrations/segmentio/lib/index.js

```diff
@@ -331,16 +332,18 @@ Segment.prototype.normalize = function(message) {
       }
 
       for (var j = 0; j < maybeBundledConfigIds[name].length; j++) {
+        bundledIdsComputedAtRuntime = true;
```

Comment:
One theory I have is that bundledConfigIds was previously exposed in renderer snippet and may be stale and overriding the values here https://github.com/segmentio/ajs-renderer/pull/256/files#diff-ca733fb7a822baffec24f4f426ccb07fce89f5b4d5856f848be4845d1d40e12bL61

Reply:
This could happen if addBundledMetadata is false